app/views/posts/index.html.erb

```diff
@@ -1,7 +1,31 @@
-<h1>Post List</h1>
+<div class="posts-container">
 
-<% @posts.each do |post|%>
-    <div class="post">
-        <%= post.text %>
+    <div class="user-box">
+        <img class="user-img" src=<%= @user.photo%> alt="user"/>
+        <div class="user-info-box">
+            <h2 class="user-name"><%= @user.name %></h2>
+            <p class="user-post-count">Number of posts: <%= @user.posts_counter%></p>
+
+        </div>
     </div>
-    <% end %>
+
+    <% @posts.each do |post| %>
+        <div class="user-recent-post">
+            <h2 class="user-post-title">Post: #<%= post.id %></h2>
+            <p class="user-post-text"><%= post.text %></p>
+            <h3 class="counter">Comments: <%= post.comments_counter%>, Likes: <%= post.likes_counter %><h3>
+
+            <div class="comment_container">
+                <% post.recent_comments.each do |comment|%>
+                    <h3 class="comments"><%= comment.author.name %>:     <%= comment.text %></h3>
+                <% end %>
+            </div>
```

Comment:
- [ ] Kindly follow the wireframe designs, this is clearly outlined in the project requirements. We don't have to do a lot of CSS, but making the pages look like the wireframes is one of the requirements.

![design](https://github.com/AbdimulhinYussuf3675/Blog-app/assets/53124548/4adebcd7-35c5-4563-bc2f-f93bd043cb74)